src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -242,12 +243,19 @@ const formConfig = {
           uiSchema: uploadPtsdDocuments.uiSchema,
           schema: uploadPtsdDocuments.schema,
         },
-        newUnemploymentFollowUp: {
-          title: 'File a Claim for Individual Unemployability',
-          path: 'new-disabilities/unemployment-status',
+        newUnemployabilityFollowUp: {
```

Comment:
`newUnemployabilityFollowUp` seems like the page that would show up _after_ we asked the veteran if they're unemployed, rather than the page that has the unemployment question itself. Is there a different name that might be more suited?

Reply:
Agreed. Naming is harddd. Maybe something like `unemployabilityStatus`?

Reply:
`unemployabilityStatus` sounds great! :+1: